### DISCUSSION TITLE
Valid Provisioning Profile Not Found

### DISCUSSION THREAD

**Discussion by Bruce9009** [AvaloniaUI/Avalonia#14531] [Q&A]
Using Visual Studio 2022 on Windows 11 and following the publish [steps](https://learn.microsoft.com/en-us/dotnet/maui/ios/deployment/publish-app-store?view=net-maui-8.0&tabs=vs) for MAUI, I am able to create a *.ipa file but I am not able to run it on iPhone because I get the following error:
`A valid provisioning profile for this executable was not found`
I also found a similar [question](https://stackoverflow.com/questions/75240454/net-maui-valid-provisioning-profile-issue) posted on Stack Overflow, but it still does not work although I tried all solutions from the post.
Does anyone have any suggestions about how to solve this question?

**Comment by maxkatz6** [comment]
The easiest way to add dev provisioning profile I know is to use Rider feature "Open project in XCode" (in the .NET project context menu). You can create provisioning profile in the XCode project settings.

Alternatively, without Rider, you can create an empty ObjC project with the same App ID, and make sure provisioning profile is set up.

**Reply by maxkatz6** [reply]
Just noticed that you use Visual Studio and Windows - no idea about this setup. Publishing iOS app on Windows isn't an easy task.
